### ISSUE TITLE
[Dropdown] Trigger misbehaving

### ISSUE THREAD

**Issue by dolanske** [dolanske/vui#27]
There is a minimum width of the trigger, which should not be there. It should be possible to trigger dropdown from 1 pixel wide block. The min-width should only be applied to the dropdown itself

**Comment by dolanske** [comment]
Add a `name` attribute to dropdown trigger
